.ci/targets/android_platform_tests_api_33.yaml

```diff
@@ -0,0 +1,29 @@
+# Same as android_platform_tests.yaml with only packages currently requiring
+# Android 33 due to test failures caused by running on Android 34 AVDs.
+tasks:
+  - name: prepare tool
+    script: .ci/scripts/prepare_tool.sh
+    infra_step: true # Note infra steps failing prevents "always" from running.
+  - name: download Dart and Android deps
+    script: script/tool_runner.sh
+    infra_step: true
+    args: ["fetch-deps", "--android", "--supporting-target-platforms-only"]
+  - name: build examples
+    script: script/tool_runner.sh
+    args: ["build-examples", "--apk"]
+  - name: lint
+    script: script/tool_runner.sh
+    args: ["lint-android"]
```

Comment:
Since these tasks aren't being passed either of the include/exclude flags, they will all run twice: once on the 34 shards and once on the 33 shards, even though there's no difference.

We should either pass the flags to everything, to keep all the tasks for a given plugin together, and running only in one place, or we should remove these two tasks from the 33 version, so all builds and lints run on the 34 shards.

I would vote for the former, for consistency, but either should work.